nominees/a11y.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "a11yproject",
+    "description": "The A11Y Project is a community-driven effort to make digital accessibility easier.",
+    "website": "https://www.a11yproject.com/",
+    "license": [{
+        "spdx": "Apache-2.0",
+        "licenseURL": "https://github.com/a11yproject/a11yproject.com/blob/main/LICENSE-APLv2"
+    }],
+    "SDGs": [{
+            "SDGNumber": 8,
```

Comment:
@SondosAhmed for `evidenceURL` instead of referencing the owner's website, try as much as possible to point to external blogs or news articles. Here is a good [one](https://www.accessibility-services.co.uk/2018/12/17/what-is-the-a11y-project/), feel free to locate others for SDGs 9 & 12. Thanks